spec/models/like_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe '#update_post_comments_counter' do
+    it 'updates the post comments_counter attribute' do
+      user = User.create(name: 'Tom')
+      post = Post.create(title: 'Hello', author: user)
+      comment = Comment.create(author: user, post:)
+
+      comment.update_post_comments_counter
+
+      expect(post.reload.comments_counter).to eq(1)
+    end
+  end
```

Comment:
- [ ] Same thing here. Kindly look at it and fix.

### Test Result

![likes](https://github.com/adarachel/rails-blog/assets/26597432/978b9343-0989-4831-96cf-bf6dafddf802)